### DISCUSSION TITLE
since v4.4.3 it doesnt work on old android tablet running chrome

### DISCUSSION THREAD

**Discussion by gonmag8** [slipx06/sunsynk-power-flow-card#211] [General]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Hi.
Not sure what changed but with version 4.4.3 My old android tablet running chrome was able to display the card.
Since updating it just says:

**Custom element doesn't exist: sunsynk-power-flow-card.**

I was trying to revert back to version 4.4.3 and it doesn't let me, the earlier I can go is 4.5.1 (and that also doesn't work)

Is there anything I can do to get v4.6 working ? if not , what is the best method to go back to 4.4.3 (didnt wanna have to restore the entire Home Assist?)

Thanks for you brilliant work, I use it everyday 👍 

### Expected behaviour

.

### Card Version

4.6 vs 4.4

### Home Assistant Version

2023.12.3

**Comment by slipx06** [comment]
Hi. There haven't been any changes since 4.4.3 that would cause this. It sounds like a browser caching issue. Have you tried clearing your browser cache (cookies, files, images etc). You can also try running it in incognito mode as a test or try using a different browser?
